frontend_lib/src/component/CardPopup/CardPopupCreateContent.jsx

```diff
@@ -73,7 +73,7 @@ export const PopupCreateContent = (props) => {
                     backgroundColor: color(props.customColor).darken(0.15).hex()
                   }
                 }}
-                disabled={!props.contentName || props.contentName.length === 0}
+                disabled={(!props.contentName || props.contentName.length === 0) && !props.isPublication}
```

Comment:
This seems too specific to me. Even if it is the only use case at the time, maybe we could change it to `allowEmptyTitle` or something like that ?

Reply:
agreed

Reply:
done